editor/gl/program.ts

```diff
@@ -58,7 +65,9 @@ export class Program {
         for (let i = 0; i < nb_uniforms; ++i) {
             let uniform_info = gl.getActiveUniform(program, i);
             if (uniform_info) {
-                let loc = gl.getUniformLocation(program, uniform_info.name);
+                // This can't be null as the uniform_info is provided by
+                // the gl context. So we assert that.
```

Comment:
That was my understanding. But software sometimes breaks, right? So my initial idea was to embedded this assumption within the code so if for one reason or another, this assumption does not hold anymore, we do not have to fight to understand what is going one.

However, if you say it is too complicated for this point in particular, it’s enough to me.